nautobot/core/views/generic.py

```diff
@@ -123,7 +126,7 @@ def get(self, request, *args, **kwargs):
                 "object": instance,
                 "verbose_name": self.queryset.model._meta.verbose_name,
                 "verbose_name_plural": self.queryset.model._meta.verbose_name_plural,
-                "changelog_url": self.get_changelog_url(instance),
+                "changelog_url": instance.get_changelog_url(),  # TODO: Remove in 2.0. This information can be retrieved from the object itself now.
```

Comment:
This is going to blow up if `instance` doesn't have a `get_changelog_url` method.

Reply:
This is probably what's causing CI to fail. This was quick and I wanted to get eyes out there. I'm going to do some proper validation.

Reply:
👍